### DISCUSSION TITLE
API Client

### DISCUSSION THREAD

**Discussion by shivamvijaywargi** [lmsGenie/lms-app#88] [Ideas]
I recently remembered about https://hoppscotch.io/ this is a free client as of now, @sub1120 please check this once and let me know if we can use this or if Insomnia is good enough for this project for everyone to be on track.

**Comment by sub1120** [comment]
> I recently remembered about https://hoppscotch.io/ this is a free client as of now, @sub1120 please check this once and let me know if we can use this or if Insomnia is good enough for this project for everyone to be on track.

Sure, I will explore this and let you know.

**Reply by sub1120** [reply]
@shivamvijaywargi , it is very nice. Let's use it. I have sent you invite for workspace. Please join. Igonore the visioncalra one, that was mistake.

**Comment by sub1120** [comment]
@shivamvijaywargi , I have invited you for cloudinary and monogodb as project owner, please accept. We will use there API keys for prod environment later.
